sql_script/MS3_insertions.sql

```diff
@@ -1,41 +1,42 @@
-/*INSERTIONS___
-#Add new inventory types to the inventory_type table*/
-INSERT
-INTO inventory_type(type_name, height, width, length)
-VALUES (new_type_name, new_height, new_width, new_length);#impact on dimensions table?
-
-/*Add a new recipient to the recipient table*/
-INSERT
-INTO recipient(id, street_address, postal_code)
+/*Add new inventory types to the inventory_type table 
+-> NOTE: only allows insertion when dimensions already exist in dimensions table*/
```

Comment:
Implement check in back-end and insert tuple in dimensions table if necessary.

Reply:
You'll want to run this query beforehand:

```sql
INSERT INTO dimensions
VALUES (new_height, new_width, new_length)
ON CONFLICT (height, width, length) DO NOTHING;
```